StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1603DocumentationMustContainValidXml.cs

```diff
@@ -45,7 +50,103 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterSyntaxNodeAction(HandleXmlElement, SyntaxKind.XmlElement);
+            context.RegisterSyntaxNodeAction(HandleXmlEmptyElement, SyntaxKind.XmlEmptyElement);
+            context.RegisterSyntaxNodeAction(HandleXmlCDataSection, SyntaxKind.XmlCDataSection);
+        }
+
+        private void HandleXmlElement(SyntaxNodeAnalysisContext context)
+        {
+            var xmlElementSyntax = context.Node as XmlElementSyntax;
+
+            if (xmlElementSyntax != null)
+            {
+                if (xmlElementSyntax.StartTag.LessThanToken.IsMissing)
+                {
+                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, xmlElementSyntax.StartTag.GetLocation(), "Xml element start tag is missing a '<'."));
+                }
+
+                if (xmlElementSyntax.StartTag.GreaterThanToken.IsMissing)
+                {
+                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, xmlElementSyntax.StartTag.GetLocation(), "Xml element start tag is missing a '>'."));
+                }
+
+                if (xmlElementSyntax.EndTag.LessThanSlashToken.IsMissing)
```

Comment:
I believe the end tag can be missing altogether:

```csharp
/// <summary>Foo
void Foo() { }
```

If that is the case, I would report the warning on the start tag with the following message:

> The XML tag '{xmlElementSyntax.StartTag.Name}' is not closed.